README.md

```diff
@@ -4,12 +4,20 @@ Library that allows you to create Spring Boot services to deploy on
 [custom-plugin-java](https://github.com/mia-platform/custom-plugin-java).
 
 ## Usage
-### Route controller
+### Route controllers
 In order to handle HTTP requests, you have to create a class that extends `CPController`.
 ``` java
     public class HelloWorldController extends CPController {
         // Endpoints defined here
     }
+
+```
+### Status routes
+In order to make status routes available, **only** your root mapped controller has to extend `CPStatusController`.
```

Comment:
I would not say its the RootController, shouldn't it extend CPController?

I'd say that **only** one controller has to extend CPStatusController

```suggestion
In order to make status routes available you can extend `CPStatusController`, please note that **only** one controller has to extend this class, otherwise Springboot will throw a runtime error due to double route registration.

Since `CPStatusController` extends `CPController` we suggest to extend this in your main controller and then extend `CPController` in other controllers.
```